docs/content/doc/installation/from-binary.en-us.md

```diff
@@ -96,7 +96,17 @@ export GITEA_WORK_DIR=/var/lib/gitea/
 cp gitea /usr/local/bin/gitea
 ```
 
-### Create service file to start gitea automatically
+## Running Gitea
+
+After the above steps, two options to run Gitea are:
```

Comment:
Would be nice to note that it is recommended to run using service

Reply:
I added `(recommended)` to the service file option now, is that clear enough or should it be added elsewhere?

Reply:
Maybe switch order so that recommended one  would be first

Reply:
Done